embark-ui/.npmignore

```diff
@@ -3,6 +3,9 @@
 .eslintrc
 config
 node_modules
+npm-shrinkwrap.json
+package-lock.json
 public
 scripts
 src
+yarn.lock
```

Comment:
`yarn.lock` doesn't need to go in the tarball published to the registry, since yarn never takes it into account when installing a package with `yarn add` or as listed in a `package.json` file.